frontdesk.tasks.scan_package_for_viruses raised unexpected: AssertionError("Wrong type for [port], should be an int [was <class 'str'>]",)

```
2/2/2017 6:09:27 PM[2017-02-02 18:09:27,547: ERROR/PoolWorker-1] Task frontdesk.tasks.scan_package_for_viruses[1014beb7-0e27-4d30-b164-3a647aca42d2] raised unexpected: AssertionError("Wrong type for [port], should be an int [was <class 'str'>]",)
2/2/2017 6:09:27 PMTraceback (most recent call last):
2/2/2017 6:09:27 PM  File "/usr/local/lib/python3.5/site-packages/celery/app/trace.py", line 367, in trace_task
2/2/2017 6:09:27 PM    R = retval = fun(*args, **kwargs)
2/2/2017 6:09:27 PM  File "/usr/local/lib/python3.5/site-packages/celery/app/trace.py", line 622, in __protected_call__
2/2/2017 6:09:27 PM    return self.run(*args, **kwargs)
2/2/2017 6:09:27 PM  File "/app/frontdesk/tasks.py", line 58, in scan_package_for_viruses
2/2/2017 6:09:27 PM    is_infected, details = utils.scan_file_for_viruses(package.file)
2/2/2017 6:09:27 PM  File "/app/frontdesk/utils.py", line 86, in scan_file_for_viruses
2/2/2017 6:09:27 PM    host=CLAMAV_HOST, port=CLAMAV_PORT)
2/2/2017 6:09:27 PM  File "/usr/local/lib/python3.5/site-packages/pyclamd/pyclamd.py", line 696, in __init__
2/2/2017 6:09:27 PM    assert isinstance(port, int), 'Wrong type for [port], should be an int [was {0}]'.format(type(port))
2/2/2017 6:09:27 PM AssertionError: Wrong type for [port], should be an int [was <class 'str'>]
```

--- a/config/settings/common.py
+++ b/config/settings/common.py
@@ -247,5 +247,5 @@ ADMIN_URL = r'^admin/'
 # Your common stuff: Below this line define 3rd party library settings
 # ClamAV Antivirus
 CLAMAV_HOST = env('CLAMAV_HOST', default='clamav')
-CLAMAV_PORT = env('CLAMAV_PORT', default=3310)
+CLAMAV_PORT = env.int('CLAMAV_PORT', default=3310)
 

--- a/frontdesk/utils.py
+++ b/frontdesk/utils.py
@@ -83,7 +83,7 @@ def scan_file_for_viruses(file):
     """
     try:
         antivirus = pyclamd.ClamdNetworkSocket(
-                host=CLAMAV_HOST, port=CLAMAV_PORT)
+                host=CLAMAV_HOST, port=int(CLAMAV_PORT))
     except pyclamd.ConnectionError as exc:
         raise AntivirusConnectionError() from exc
 

